src/en/discover/technology/index.html

```diff
@@ -42,10 +37,10 @@ <h2 class="color-white h1">Infinitely scalable data storage for object, block an
 
 <section class="section">
   <div class="max-w-192 mx-auto text-center">
-    <h2 class="h2">Store Data in the Cloud</h2>
+    <h2 class="h2">The Ceph Stack -- Architectural Overview</h2>
+    <a href="#object"><img alt="" src="/assets/stack.png" /></a>
```

Comment:
@zdover23 This link and image was introduced to the page here. What was the desired outcome here? Is this needed in the new site? What is it attempting to link to? Has the asset been added to our `/src/assets/bitmaps` directory?

Reply:
> @zdover23 This link and image was introduced to the page here. What was the desired outcome here? Is this needed in the new site? What is it attempting to link to? Has the asset been added to our `/src/assets/bitmaps` directory?

@adamduncan: I think I've got it fixed here: https://github.com/ceph/ceph.io/pull/177